ui/pages/confirmation/templates/flask/snap-confirmation/snap-confirmation.js

```diff
@@ -19,46 +18,7 @@ function getValues(pendingApproval, t, actions) {
           props: {
             snapName,
           },
-          children: [
-            {
-              element: 'Typography',
-              key: 'title',
-              children: title,
-              props: {
-                variant: TYPOGRAPHY.H3,
-                fontWeight: 'bold',
-                boxProps: {
-                  marginBottom: 4,
-                },
-              },
-            },
-            ...(description
-              ? [
-                  {
-                    element: 'Typography',
-                    key: 'subtitle',
-                    children: description,
-                    props: {
-                      variant: TYPOGRAPHY.H6,
-                      boxProps: {
-                        marginBottom: 4,
-                      },
-                    },
-                  },
-                ]
-              : []),
-            ...(textAreaContent
-              ? [
-                  {
-                    element: 'Copyable',
-                    key: 'snap-dialog-content-text',
-                    props: {
-                      text: textAreaContent,
-                    },
-                  },
-                ]
-              : []),
-          ],
+          children: mapToTemplate(content),
```

Comment:
Does this need to be an array?

Reply:
No, it can also be an object if the children is a single component. Ref: https://github.com/MetaMask/metamask-extension/blob/f321225ed409c7502b15d295c6dc4ba09200731c/ui/pages/confirmation/templates/flask/snap-prompt/snap-prompt.js#L30